schema/list.json

```diff
@@ -24,6 +24,6 @@
             }
         },
         "required": ["id", "name"],
-        "additionalProperties": false
```

Comment:
Because the `Packet` model has been extended, some extra properties are now being returned by the index endpoint (`files` and `depends`). At the moment the consumer of this endpoint (Packit) has no use for this info; this endpoint is just used to populate a more minimal index table. So the question is, should we introduce a separate struct (`PacketMinimal` or `PackitIndexItem` or something) to keep the index data returned more minimal?

Reply:
Another consideration is that at the moment, the logic to retrieve packet metadata is shared between the `/packet/metadata` and the querying and the latter will need the extra properties. So changing the model will also require duplicating that logic to some extent, just returning a more minimal struct in one case.

Reply:
I think we'll want separate schemas - one that is the exact properties as other outpack implementations (particularly as we build out more of what the rust version can do) - and others that are tailored to packit (or other internal uses).